pages/api/users/create.js

```diff
@@ -68,8 +69,7 @@ export default async (req, res) => {
     salt,
     username: newUsername,
     data: {
-      photo:
-        "https://slate.textile.io/ipfs/bafkreiexygfz4e5resu66xfviokddariztq4onuai5wii5mdd7syshftca",
+        photo: SlateManager.getRandomSlateElement({id: Environment.AVATAR_SLATE_ID}),
```

Comment:
catch a failure case here so if people don't have env variables it will use the old case